windtunnel/windtunnel.py

```diff
@@ -47,21 +47,73 @@
 class WindTunnel:
     """WindTunnel scenario."""
 
-    def __init__(self, inputs_dir: str = "./inductiva_input/"):
-        """Initializes the `WindTunnel` scenario object.
+    def __init__(self, dimensions: tuple[int, int, int] = (20, 10, 8)):
+        """
+        Initializes the wind tunnel with given dimensions.
+
+        Parameters:
+        - dimensions: A tuple (length, width, height) for the tunnel size.
         """
```

Comment:
consider adding here for readability:
```python
length, width, height = dimensions
```

and then use those variables in the walls definitions...